packages/dev/core/src/Gizmos/planeRotationGizmo.ts

```diff
@@ -122,34 +122,45 @@ export class PlaneRotationGizmo extends Gizmo implements IPlaneRotationGizmo {
             vUV = uv;
         }`;
 
-    protected static _RotationGizmoFragmentShader = `
-        precision highp float;
-        varying vec2 vUV;
-        varying vec3 vPosition;
-        uniform vec3 angles;
-        #define twopi 6.283185307
-        void main(void) {
-            vec2 uv = vUV - vec2(0.5);
-            float angle = atan(uv.y, uv.x) + 3.141592;
-            float delta = gl_FrontFacing ? angles.y : -angles.y;
-            float begin = angles.x - delta * angles.z;
-            float start = (begin < (begin + delta)) ? begin : (begin + delta);
-            float end = (begin > (begin + delta)) ? begin : (begin + delta);
-            float len = sqrt(dot(uv,uv));
-            float opacity = 1. - step(0.5, len);
-
-            float base = abs(floor(start / twopi)) * twopi;
-            start += base;
-            end += base;
-
-            float intensity = 0.;
-            for (int i = 0; i < 5; i++)
-            {
-                intensity += max(step(start, angle) - step(end, angle), 0.);
-                angle += twopi;
+    protected static _createRotationGizmoFragmentShader(color: Color3) {
+        let r = `${color.r}`;
+        let g = `${color.g}`;
+        let b = `${color.b}`;
+
+        r = r.includes('.') ? r : `${r}.`;
+        g = g.includes('.') ? g : `${g}.`;
+        b = b.includes('.') ? b : `${b}.`;

```

Comment:
This feels a bit a lot of code to achieve having always a "." in the color :innocent:.

Does maybe someone have a better idea?